src/components/calcite-tooltip/calcite-tooltip.scss

```diff
@@ -74,6 +74,7 @@ slot[name="image"]::slotted(img) {
   border: none;
   display: block;
   cursor: pointer;
+  border-radius: 2px;
```

Comment:
Sorry I wasn't super clear, only the border that is adjacent to the container's 2px border should have this 2px border radius applied (and reciprocal for rtl) - in alerts (no top-right because the accent border has it) https://github.com/Esri/calcite-components/blob/e8c60b9dca425f2c33eb987e242ff48129cf27a7/src/components/calcite-alert/calcite-alert.scss#L162

Seems like we'd need to detect if an image is present to decide when to apply this. 
![Group 3](https://user-images.githubusercontent.com/4733155/66350287-c4f9d780-e90f-11e9-8c20-97aa13323509.jpg)